docs/helpers.md

```diff
@@ -0,0 +1,25 @@
+> Helper Classes
+
+# Usefull warp helper classes
+
+## page container
+The `.page-container` helper class job is to normalise page behaviour across the pages/sites, most pages should be set up with this.
+
+- Sets a max-width for the page ( This is dictated by the banner formats )
+- Left aligns the page if there is room for banners on both sides
+- Centers the page when there is room for banners on both sides.
+- Sets up spacing between content and banners ( Also calculated from banner sizes )
+
+Header/Footer also use the same helper class to follow the same pattern.
+
+<container>
+  <div class="flex justify-center">
+    <a href="/css-docs/page-container.png" alt="Open picture in a new window"><img src="/page-container.png" class="object-cover" alt="example of what page-container does as an overlay ontop of finn.no" /></a>
```

Comment:
Would it make sense to open it in a different tab, or as a modal (like the icons)? I'm saying that only based on my experience when testing the page 😄 When I clicked the image I saw a full screen version of it, which looked as if it was presented in a modal with black backdrop. Because of that I found it difficult to figure out how to "close" that full screen view. Only after closing the whole tab in the browser did I realise that the image was in fact on a new url path within the same tab as the css-docs. So now I understand that I had to use browser's "go back" button to return to the page container docs. If you think this is fine, then ignore my suggestion :) 
<img width="1555" alt="Screenshot 2023-12-01 at 12 35 36" src="https://github.com/warp-ds/css-docs/assets/41303231/6aa59673-be36-4cfc-8a62-848919f1ca12">

Reply:
Sure, thats a good point! Can someone help me implement the modal in those docs? I dunno how to get them running with correct styling inside of this vitepress installation.

Reply:
It could be dark mode issues need to be fixed first here anyway for a modal to be styled correctly ([WARP-402](https://nmp-jira.atlassian.net/browse/WARP-402)), so if you want to look into implementing a modal here, maybe it makes sense to do it separately from this PR.